rele/retry_policy.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class RetryPolicy:
+    """A RetryPolicy object encapsulates the validation rules.
+
+    Defines retry policy settings and ensures the values are correct.
+    If provided values are wrong, a ValidationError is raised.
+
+    :param minimum_backoff: int Accepts values greater than 0
+    :param maximum_backoff: int Accepts values greater than minimum_backoff.
+    """
+
+    minimum_backoff: None
```

Comment:
```suggestion
    minimum_backoff: Optional[Int]
```
Should this be an int?

Reply:
Yep, but not optional. Thanks!